address

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
Is this what a symlink looks like in a PR?  I don't think I've ever had to know :-)